src/golang/lib/workflow/operator/operator.go

```diff
@@ -98,6 +98,7 @@ func NewOperator(
 	execEnv *exec_env.ExecutionEnvironment,
 	aqPath string,
 	db database.Database,
+	engineJobManager *job.JobManager,
```

Comment:
Can you add a function comment here that `engineJobManager` serves as an override and is there only so that context can be shared between operators?

Reply:
Should we call it `dagJobManager` instead? Instead it is scoped to the dag. Not sure what value the word "engine" contributes.

Reply:
Yep, I like dagJobManager as well!